Dockerfile.protofix

```diff
@@ -0,0 +1,21 @@
+FROM golang:latest
+MAINTAINER derek@pachyderm.io
+
+RUN apt-get update -yq
+RUN apt-get install -yq unzip
+
+RUN wget https://github.com/google/protobuf/releases/download/v3.0.0-beta-2/protoc-3.0.0-beta-2-linux-x86_64.zip -O protoc.zip
+RUN unzip protoc.zip -d /bin
+
+RUN go get -u -v github.com/golang/protobuf/proto
+RUN go get -u -v github.com/golang/protobuf/protoc-gen-go
+RUN go get -u github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway
+RUN go get -v go.pedge.io/protoeasy/cmd/protoeasy
+
+CMD \
+    cd $GOPATH/src/github.com/pachyderm/pachyderm && \
+    go install ./src/server/cmd/protofix && \
+    rm -rf src/server/vendor && \ 
+    protoeasy --grpc --go --go-import-path github.com/pachyderm/pachyderm/src src && \
+    protofix fix src && \
+    git checkout src/server/vendor
```

Comment:
I think it's a little weird to put this as part of the Dockerfile. Could we keep it in the Makefile?